src/modules/org-creator/orgCreator.service.ts

```diff
@@ -45,7 +45,7 @@ export class OrgCreatorService {
     }
 
     const { claimData } = jwt.decode(token) as IClaimToken;
-
+    this.logger.logger(`claim data decoded: ${JSON.stringify(claimData)}`);
```

Comment:
Should this be `.log`?
```suggestion
    this.logger.log(`claim data decoded: ${JSON.stringify(claimData)}`);
```

Reply:
Yep, not all changes were applied. Fixed